### DISCUSSION TITLE
How to use ILSpy on Linux system?

### DISCUSSION THREAD

**Discussion by vit1251** [icsharpcode/ILSpy#2926] [General] [answered]
During research ILSpy version 7.0 with Mono project I meet with error

```
The entry point method could not be loaded due to Could not load file or assembly 'PresentationFramework, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies.
```

Since I reading Mono about PresentationFramework warning suggest using Xamarin or WidnowsForms instead. My question about using ILSpy on Linux platform?

**Comment by christophwille** [comment, marked as answer]
On Linux, you can either use ilspycmd or AvaloniaILSpy.
